.drone.yml

```diff
@@ -1,78 +1,469 @@
 ---
 kind: pipeline
-name: default
+name: testing
 
 platform:
   os: linux
   arch: amd64
 
-workspace:
-  base: /go
-  path: src/github.com/drone/autoscaler
-
 steps:
+- name: vet
+  pull: always
+  image: golang:1.11
+  commands:
+  - go vet ./...
+  environment:
+    GO111MODULE: on
```

Comment:
remove `GO111MODULE: on` since it is not necessary when building outside the gopath, and remove `pull: always` because it prevents me from using `drone exec` locally when I don't have internet (like when I'm working from a coffee shop). This same feedback should apply to all official Drone repos in the `drone/` org. You can use slightly different standard for `drone-plugins/` as you see fit.

Reply:
Will fix that for all the prs